### DISCUSSION TITLE
Bot Verification System

### DISCUSSION THREAD

**Discussion by FabrizioCoder** [stoatchat/discussions#317] [Feature Suggestions]
Hi, I created this discussion because I didn't know where else to give the suggestion haha

As you have read in the title, I suggest you add a bot verification system.

Some requirements could be:

- 100 Servers.
-  That the bot complies with all the rules of the application.

Thanks for your attention

**Comment by erkinalp** [comment]
How would verification work in this case? Remember revolt.chat is not a platform, it is an open source protocol for instant messaging and real time communication.

**Reply by Zomatree** [reply]
Revolt is a platform not just a protocol, verification could be added but it isn't on our roadmap yet and wouldn't be a thing for quite a while.
